packages/p2p/src/utils/chat-message.js

```diff
@@ -38,3 +53,6 @@ export const convertFromChannelMessage = channel_message => {
         url: channel_message.url,
     });
 };
+
+export const admin_message =
+    "Hello! This is where you can chat with the counterparty to confirm the order details.\nNote: In case of a dispute, we'll use this chat as a reference.";
```

Comment:
Don't we need to translate this text?

Reply:
no. since it is a sendbird message, it doesn't work with localize. so for now, translations are not supported.

Reply:
But, the message comes from our end right. So, why not we translate it first and send the translated message to sendbird?

Reply:
that doesn't work here. because the message is not handled from our end. sendbird do has some internal translation tool available, which i believe is an extra paid thing.. but which is now out of scope. https://sendbird.com/docs/chat/v4/javascript/message/translating-messages/auto-translate-messages.